docs/index.rst

```diff
@@ -13,7 +13,27 @@ Getting Started
 Installation
 ------------
 
-Install it with pip or conda.
+Testing
++++++++
+
+If you are testing a pre-release version of stellarphot we recommend setting up
+a virtual environment and installing stellarphot in this environment.
+
+Creating an environment with `conda` or `mamba`:
+
+```bash
+mamba create -n stellarphot-test python=3.11
```

Comment:
So does stellarphot-test have all the dependencies defined?  I have my own testing environment for stellarphot under conda and I should probably compare the two setups.

Reply:
No -- this section should be modified/removed. For testing of this branch I sent the instructions below. It creates a bare-bones environment then `pip` installs everything from a github URL so people can try out the UI independent of whether we've done a pre-release or not and dodging all of the intricacies of git:

---
This first look will be to try out a couple of jupyter notebook-based forms for entering a bunch of settings.

I’d recommend, for now, that when you are testing you make a python environment for testing in, the delete the environment when you are done instead of trying to frequently update stellarphot. Instructions for doing that if you use conda are below, happy to provide non-conda instructions too.

To make an environment for testing called sp-test:

```
conda create -n sp-test python=3.11
```

When the environment is created, activate it:

```
conda ativaste sp-test
```

The command to install the version of stellarphot to try this time around is:

```
pip install git+https://github.com/mwcraig/stellarphot.git@ui-work
```

When this is done, please save the attached notebook to a folder, and navigate into that folder in the terminal. Run “`jupyter lab`” to start up jupyter and try going through the notebook. After several cells of set up it will present forms for you to enter settings. Don’t worry about using realistic values for now.

I’m expecting you will have questions and that it needs more instructions, but figured getting a quick look sooner was better than waiting longer.

When you are done you can deactivate the environment and then remove it:

```
conda deactivate
conda remove --all --yes sp-test
```